docs/developer/clients.md

```diff
@@ -2,237 +2,29 @@
 
 Client libraries enable easy access to drand randomness in specific programming languages. They also provide piece of mind by _verifying_ randomness rounds and add additional features like failover, racing, aggregation, and caching.
 
-Drand randomness is available over HTTP, [gRPC](https://grpc.io/) and [libp2p PubSub](https://docs.libp2p.io/concepts/publish-subscribe/), but availability of transports other than HTTP in client libraries is implementation dependent.
+Drand randomness is available over [HTTP](/developer/http-api/), [libp2p PubSub](/developer/gossipsub/) and gRPC.
 
-Choose a drand client library that matches the language in which your application is written:
+Currently, the following client libraries are available:
 
 - [Go](#go)
 - [JS](#js)
 
-Language not listed? You can make raw requests to the [HTTP API](/developer/http-api/). If you have created a client library or know of a client library not listed here, [please let us know by opening an issue](https://github.com/drand/website/issues/new).
+If your application cannot use the libraries above, you can still use the [drand-client CLI](/developer/drand-client) or make requests to the [HTTP API](/developer/http-api/) endpoints. If you have created a client library or know of a client library not listed here, [please let us know by opening an issue](https://github.com/drand/website/issues/new).
 
 ## Go
 
-### Install
-
-```sh
-go get github.com/drand/drand
-```
-
-### Usage
-
-[API reference on pkg.go.dev](https://pkg.go.dev/github.com/drand/drand/client)
-
-The drand client library is structured with a base client interface in `/client`, and with protocol-specific transport implementations in `/client/http`, `/client/grpc` and `/lp2p/client`. The main `client` package holds transport agnostic logic for retrying, validation, and caching.
+The Go drand client library is structured with a base client interface in `/client`, and with protocol-specific transport implementations in `/client/http`, `/client/grpc` and `/lp2p/client`. The main `client` package holds transport agnostic logic for retrying, validation, and caching.
 
-```go
-package main
+For documentation and examples please check the Go clients reference:
 
-import (
-	"context"
-	"encoding/hex"
-	"fmt"
-	"github.com/drand/drand/client"
-)
-
-var chainHash, _ = hex.DecodeString("8990e7a9aaed2ffed73dbd7092123d6f289930540d7651336225dc172e51b2ce")
-
-func main() {
-	c, err := client.New(
-		client.From(/* add concrete client implementations here (see below) */),
-		client.WithChainHash(chainHash),
-	)
-
-	// e.g. use the client to get the latest randomness round:
-	r := c.Get(context.Background(), 0)
-
-	fmt.Println(r.Round(), r.Randomness())
-}
-```
```

Comment:
Not having any typical usage examples here makes this page significantly less interesting to read. I agree they should be in godoc but why not trim this down to just a single example using HTTP that'll be used by most people, that can just be copy/pasted?

Reply:
I'm against maintaining Go documentation outside of Godocs because they are exactly for that. They can copy-paste the example from Godocs. Users should be taught to go to godocs pages and read things there if that is not their first impulse.